src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -52,16 +52,27 @@ public function __construct(FilesystemInterface $driver)
      * Assert that the given file exists.
      *
      * @param  string|array  $path
+     * @param  string|null  $content
      * @return $this
      */
-    public function assertExists($path)
+    public function assertExists($path, $content = null)
```

Comment:
This is a breaking change unfortunately :-/

Reply:
Indeed - but I think this may be a breaking change that doesn't "really" affect people.

Reasoning: This method seems to be used only in testing, and people would have to extend this class, and this method, for this to affect their test suites.

What do you think @driesvints ?

Reply:
I personally feel like we should avoid all breaking changes. Every one is a risk we take. But let's let @taylorotwell decide.